src/tools/ceph_kvstore_tool.cc

```diff
@@ -347,6 +348,8 @@ int main(int argc, const char *argv[])
     st.compact_range(prefix, start, end);
   } else if (cmd == "stats") {
     st.print_stats();
+  } else if (cmd == "dissect") {
+    st.dissect();
```

Comment:
I feel like "dissect" sounds unintuitive. 
How about "kv_hist[ogram]". But is is just a proposal.